src/Server/modules/DataClasses.js

```diff
@@ -119,6 +125,8 @@ module.exports = {
 					this.numDeaths[dates[i]] = parseInt(values[i]);
 				}
 			}
+
+			this.currentNumDeaths = this.numDeaths[dates[dates.length-1]];
```

Comment:
currentNumDeaths is not always the last element in the dictionary. (dates is also not sorted.)

Reply:
changed it to:
this.currentNumDeaths = M.GetMostRecentValue(this.numDeaths);